pkg/util/util_test.go

```diff
@@ -0,0 +1,68 @@
+package util
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+const (
+	testAttempts = 2
```

Comment:
any reason not to test with value of `retryAttempts` since that the hardcoded value for "prod". are the tests unnecessary slower?